app/src/main/java/com/github/bgabriel998/softwaredevproject/gallery/ImageActivity.java

```diff
@@ -8,6 +8,7 @@
 import android.graphics.Matrix;
 import android.media.ExifInterface;
```

Comment:
Cirrus suggests that we avoid `android.media.ExifInterface` and we replace it with the one I wrote in the suggestion.
```suggestion
import android.support.media.ExifInterface;
```

I found [this](https://stackoverflow.com/questions/44246110/android-error-while-using-exifinterface) as an explanation of why it is better. It may be necessary to add the following line in build.gradle, as stated in the linked resource:
`compile "com.android.support:exifinterface:25.1.0"`

Reply:
According to your link it is an issue for Android versions lower than 24. We have min at 24. But I can still change it if you want ofcourse!

Reply:
Oh you're right. I don't think it is necessary then.